dias/analyzers/source_spectra_analyzer.py

```diff
@@ -353,7 +353,7 @@ def run(self):
                 src_ra, src_dec = ephemeris.object_coords(
                     src_body, date=np.median(file_timestamp), deg=False
                 )
-                source_start_time, source_stop_time = file_list[file_index][1]
+                source_start_time, source_stop_time = timestamp[0], timestamp[-1]
```

Comment:
I can't find the variable `timestamp`. Does it exist?

Reply:
My bad. It's `file_timestamp`. It's fixed now.

Reply:
Thanks! Can you do a test run to make sure this works now?